### PR TITLE
Added support for the Tajik language.

### PR DESCRIPTION
**Pull request by claudep**
Thanks Sirius Sufiew for contributing that support.

### CODE REVIEW

**Comment by claudep** [comment]
@566S, please double check this pull request!

File: django/conf/locale/__init__.py [hunk 1]
```diff
@@ -506,6 +506,12 @@
         'name': 'Telugu',
         'name_local': 'తెలుగు',
     },
+    'tg': {
+        'bidi': False,
+        'code': 'tg',
+        'name': 'Tajik',
+        'name_local': 'забо́ни тоҷикӣ́',
```
Comment (felixxm) [inline django/conf/locale/__init__.py:513]: `тоҷикӣ`?

**Comment by felixxm** [comment]
@566S Can you take a look?

**Comment by 566S** [comment]
Made you some corrections ?

**Comment by claudep** [comment]
No, but that's the first comment we receive from you on this ticket. The other ones you showed me on the screen capture never reached Github!

**Comment by felixxm** [comment]
I also left one [question](https://github.com/django/django/pull/12918#discussion_r427732013).

**Comment by 566S** [comment]
Ok.
You have to change any 
"tg" to 'tj''.
The short form(abbreviation) of Tajik is "tj' , but No "tg'

And also the value of "name_local" is "тоҷикӣ". No забони тоҷикӣ

**Comment by claudep** [comment]
According to https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes, the ISO 639-1 language code for Tajik is "tg". Cold you double check and confirm?

**Comment by 566S** [comment]
You right. According to the ISO 639-1, it is "tg". 
But it must be "tj". 
You can add as "tj". Is it possible to change it after ?

**Comment by claudep** [comment]
Sorry, but we won't go against ISO 639-1. I'll update the PR with the proper name_local ASAP.

**Comment by felixxm** [comment]
buildbot

**Pull request merged**